javascript/modules/common/test/unit/encoding.test.ts

```diff
@@ -282,7 +282,7 @@ describe("Value encoding", () => {
     ]
     for (let input of inputs) {
       const result = hexToBytes(input.in);
-      expect(result).toEqual(input.out);
+      expect(result.toString()).toEqual(input.out.toString());
```

Comment:
Are we sure that `.toString()` doesn't simply generate something like `[object Object]`?

Reply:
Yes, tested it

```ts
const a = new Uint8Array([11,12])
console.log(a.toString())
 // "11,12"
```